cf/test/test_Data.py

```diff
@@ -3391,14 +3391,12 @@ def test_Data_log(self):
         self.assertEqual(d.shape, b.shape)
 
         # Test an arbitrary base, using 4 (not a special managed case like 10)
-        # TODODASK: reinstate this assertion once mask property is
-        # daskified.
-        # a = np.array([[4, 16, 4**3.5], [0, 1, 0.25]])
-        # b = np.log(a) / np.log(4)  # the numpy way, using log rules from school
-        # c = cf.Data(a, "s")
-        # d = c.log(base=4)
-        # self.assertTrue((d.array == b).all())
-        # self.assertEqual(d.shape, b.shape)
+        a = np.array([[4, 16, 4 ** 3.5], [0, 1, 0.25]])
+        b = np.log(a) / np.log(4)  # the numpy way, using log rules from school
+        c = cf.Data(a, "s")
+        d = c.log(base=4)
+        self.assertTrue((d.array == b).all())
+        self.assertEqual(d.shape, b.shape)
```

Comment:
Might be nice to test the units:
```suggestion
        self.assertEqual(d.shape, b.shape)
        self.assertEqual(d.Units, cf.Units("1"))
```